Build failure: kdePackages.sierra-breeze-enhanced

### Nixpkgs version

- Unstable (26.05)

### Steps to reproduce

```sh
nix build nixpkgs#kdePackages.sierra-breeze-enhanced
```

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/326479137

### Relevant log output

```console
Running phase: qtPreHook
@nix { "action": "setPhase", "phase": "qtPreHook" }
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/fd6xkjrcaky07729wrklz48nf4klc63i-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/doc/sierrabreezeenhanced -DCMAKE_INSTALL_INFODIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/include -DCMAKE_INSTALL_SBINDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib -DCMAKE_STRIP=/nix/store/kilyfvfmh9x9rlaj47ph1916lbc1pnsr-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/kilyfvfmh9x9rlaj47ph1916lbc1pnsr-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/kilyfvfmh9x9rlaj47ph1916lbc1pnsr-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1 -DCMAKE_INSTALL_PREFIX=\$out -DBUILD_TESTING=OFF -DKDE_INSTALL_USE_QT_SYS_PATHS=ON -DKDE_INSTALL_EXECROOTDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1 -DKDE_INSTALL_BINDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/bin -DKDE_INSTALL_SBINDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/sbin -DKDE_INSTALL_LIBDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib -DKDE_INSTALL_LIBEXECDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/libexec -DKDE_INSTALL_CMAKEPACKAGEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib/cmake -DKDE_INSTALL_QTPLUGINDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib/qt-6/plugins -DKDE_INSTALL_PLUGINDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib/qt-6/plugins -DKDE_INSTALL_QMLDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib/qt-6/qml -DKDE_INSTALL_INCLUDEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/include -DKDE_INSTALL_LOCALSTATEDIR=/var -DKDE_INSTALL_SHAREDSTATEDIR=/com -DKDE_INSTALL_DATAROOTDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share -DKDE_INSTALL_DATADIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share -DKDE_INSTALL_DOCBUNDLEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/doc/HTML -DKDE_INSTALL_KCFGDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/config.kcfg -DKDE_INSTALL_KCONFUPDATEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/kconf_update -DKDE_INSTALL_KAPPTEMPLATESDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/kdevappwizard/templates -DKDE_INSTALL_KFILETEMPLATESDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/kdevfiletemplates/templates -DKDE_INSTALL_KXMLGUIDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/kxmlgui5 -DKDE_INSTALL_KNOTIFYRCDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/knotifications6 -DKDE_INSTALL_ICONDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/icons -DKDE_INSTALL_LOCALEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/locale -DKDE_INSTALL_SOUNDDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/sounds -DKDE_INSTALL_TEMPLATEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/templates -DKDE_INSTALL_WALLPAPERDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/wallpapers -DKDE_INSTALL_APPDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/applications -DKDE_INSTALL_DESKTOPDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/desktop-directories -DKDE_INSTALL_MIMEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/mime/packages -DKDE_INSTALL_METAINFODIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/metainfo -DKDE_INSTALL_QTQCHDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/doc/qch -DKDE_INSTALL_QCHDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/doc/qch -DKDE_INSTALL_MANDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/man -DKDE_INSTALL_INFODIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/info -DKDE_INSTALL_DBUSDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/dbus-1 -DKDE_INSTALL_DBUSINTERFACEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/dbus-1/interfaces -DKDE_INSTALL_DBUSSERVICEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/dbus-1/services -DKDE_INSTALL_DBUSSYSTEMSERVICEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/dbus-1/system-services -DKDE_INSTALL_SYSCONFDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/etc -DKDE_INSTALL_CONFDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/etc/xdg -DKDE_INSTALL_AUTOSTARTDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/etc/xdg/autostart -DKDE_INSTALL_LOGGINGCATEGORIESDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/qlogging-categories6 -DKDE_INSTALL_SYSTEMDUNITDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib/systemd -DKDE_INSTALL_SYSTEMDUSERUNITDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/systemd/user -DKDE_INSTALL_EXECROOTDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1 -DKDE_INSTALL_BINDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/bin -DKDE_INSTALL_SBINDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/sbin -DKDE_INSTALL_LIBDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib -DKDE_INSTALL_LIBEXECDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/libexec -DKDE_INSTALL_CMAKEPACKAGEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib/cmake -DKDE_INSTALL_QTPLUGINDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib/qt-6/plugins -DKDE_INSTALL_PLUGINDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib/qt-6/plugins -DKDE_INSTALL_QMLDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib/qt-6/qml -DKDE_INSTALL_INCLUDEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/include -DKDE_INSTALL_LOCALSTATEDIR=/var -DKDE_INSTALL_SHAREDSTATEDIR=/com -DKDE_INSTALL_DATAROOTDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share -DKDE_INSTALL_DATADIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share -DKDE_INSTALL_DOCBUNDLEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/doc/HTML -DKDE_INSTALL_KCFGDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/config.kcfg -DKDE_INSTALL_KCONFUPDATEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/kconf_update -DKDE_INSTALL_KAPPTEMPLATESDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/kdevappwizard/templates -DKDE_INSTALL_KFILETEMPLATESDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/kdevfiletemplates/templates -DKDE_INSTALL_KXMLGUIDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/kxmlgui5 -DKDE_INSTALL_KNOTIFYRCDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/knotifications6 -DKDE_INSTALL_ICONDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/icons -DKDE_INSTALL_LOCALEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/locale -DKDE_INSTALL_SOUNDDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/sounds -DKDE_INSTALL_TEMPLATEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/templates -DKDE_INSTALL_WALLPAPERDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/wallpapers -DKDE_INSTALL_APPDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/applications -DKDE_INSTALL_DESKTOPDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/desktop-directories -DKDE_INSTALL_MIMEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/mime/packages -DKDE_INSTALL_METAINFODIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/metainfo -DKDE_INSTALL_QTQCHDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/doc/qch -DKDE_INSTALL_QCHDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/doc/qch -DKDE_INSTALL_MANDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/man -DKDE_INSTALL_INFODIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/info -DKDE_INSTALL_DBUSDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/dbus-1 -DKDE_INSTALL_DBUSINTERFACEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/dbus-1/interfaces -DKDE_INSTALL_DBUSSERVICEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/dbus-1/services -DKDE_INSTALL_DBUSSYSTEMSERVICEDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/dbus-1/system-services -DKDE_INSTALL_SYSCONFDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/etc -DKDE_INSTALL_CONFDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/etc/xdg -DKDE_INSTALL_AUTOSTARTDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/etc/xdg/autostart -DKDE_INSTALL_LOGGINGCATEGORIESDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/qlogging-categories6 -DKDE_INSTALL_SYSTEMDUNITDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/lib/systemd -DKDE_INSTALL_SYSTEMDUSERUNITDIR=/nix/store/8sx4xgq33il9yqqvn64nz91jipr26hn9-sierra-breeze-enhanced-2.1.1/share/systemd/user
CMake Warning (dev) at CMakeLists.txt:1 (project):
  cmake_minimum_required() should be called prior to this top-level project()
  call.  Please see the cmake-commands(7) manual for usage documentation of
  both commands.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- The C compiler identification is GNU 14.3.0
-- The CXX compiler identification is GNU 14.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/kilyfvfmh9x9rlaj47ph1916lbc1pnsr-gcc-wrapper-14.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/kilyfvfmh9x9rlaj47ph1916lbc1pnsr-gcc-wrapper-14.3.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Deprecation Warning at CMakeLists.txt:5 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.


-- Looking for __GLIBC__
-- Looking for __GLIBC__ - found
-- Performing Test _OFFT_IS_64BIT
-- Performing Test _OFFT_IS_64BIT - Success
-- Performing Test HAVE_DATE_TIME
-- Performing Test HAVE_DATE_TIME - Success
-- Found OpenGL: /nix/store/39fgb5pr1r6k3szvfhpfmv9jp829kisx-libglvnd-1.7.0/lib/libOpenGL.so
-- Found WrapOpenGL: TRUE
-- Found WrapVulkanHeaders: /nix/store/4555d6l3mc79fnbcp5sckgjh30agg6sd-vulkan-headers-1.4.328.0/include
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE
-- Found KF6IconThemes: /nix/store/qs36w8xja410nznjzm1r5n924nc7jlx9-kiconthemes-6.20.0-dev/lib/cmake/KF6IconThemes/KF6IconThemesConfig.cmake (found version "6.20.0")
-- Found KF6CoreAddons: /nix/store/b4l24718ycg1453s89saf68klca8rp94-kcoreaddons-6.20.0-dev/lib/cmake/KF6CoreAddons/KF6CoreAddonsConfig.cmake (found version "6.20.0")
-- Found KF6GuiAddons: /nix/store/lgh3xsm0vmghinp3h77y86dab61c95nf-kguiaddons-6.20.0-dev/lib/cmake/KF6GuiAddons/KF6GuiAddonsConfig.cmake (found version "6.20.0")
-- Found KF6ConfigWidgets: /nix/store/5adcqgzs0s7g6xb50r7lv9m6dd90fm6x-kconfigwidgets-6.20.0-dev/lib/cmake/KF6ConfigWidgets/KF6ConfigWidgetsConfig.cmake (found version "6.20.0")
-- Found X11: /nix/store/jx7hk27qxykfanlwil7wbcz62mcz7wd5-xorgproto-2024.1/include
-- Looking for XOpenDisplay in /nix/store/6an80jglfqg4gkiyl87wxwa2zcplwn2f-libx11-1.8.12/lib/libX11.so;/nix/store/kx3mgcnyhhaz309x81gr4hjwgxn64dg8-libxext-1.3.6/lib/libXext.so
-- Looking for XOpenDisplay in /nix/store/6an80jglfqg4gkiyl87wxwa2zcplwn2f-libx11-1.8.12/lib/libX11.so;/nix/store/kx3mgcnyhhaz309x81gr4hjwgxn64dg8-libxext-1.3.6/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Found KF6WindowSystem: /nix/store/2hjwpwp7ifv8ibhzhwp2is4ca4raryxp-kwindowsystem-6.20.0-dev/lib/cmake/KF6WindowSystem/KF6WindowSystemConfig.cmake (found version "6.20.0")
-- Found Gettext: /nix/store/nzl79pryz8bzzkpq3j6l87jdiikfksqz-gettext-0.25.1/bin/msgmerge (found version "0.25.1")
-- Found KF6I18n: /nix/store/35qvln43kh7qjnd14d9jzqvb55jl6fld-ki18n-6.20.0-dev/lib/cmake/KF6I18n/KF6I18nConfig.cmake (found version "6.20.0")
-- Found KF6: success (found suitable version "6.20.0", minimum required is "6.0.0") found components: IconThemes CoreAddons GuiAddons ConfigWidgets WindowSystem I18n
-- Found XCB_XCB: /nix/store/3y1g6rqwx7xjg0pb7zp7dvvssr4n5crc-libxcb-1.17.0/lib/libxcb.so (found version "")
-- Found XCB: /nix/store/3y1g6rqwx7xjg0pb7zp7dvvssr4n5crc-libxcb-1.17.0/lib/libxcb.so  found components: XCB
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY - Success
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Success
-- Configuring done (18.1s)
CMake Warning (dev) at /nix/store/b4l24718ycg1453s89saf68klca8rp94-kcoreaddons-6.20.0-dev/lib/cmake/KF6CoreAddons/KF6CoreAddonsMacros.cmake:49 (add_library):
  Policy CMP0115 is not set: Source file extensions must be explicit.  Run
  "cmake --help-policy CMP0115" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.

  File:

    /build/source/config/kcm_sierrabreezeenhanceddecoration.cpp
Call Stack (most recent call first):
  config/CMakeLists.txt:13 (kcoreaddons_add_plugin)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Error at CMakeLists.txt:125 (target_link_libraries):
  Target "sierrabreezeenhanced" links to:

    Qt6::GuiPrivate

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Error at config/CMakeLists.txt:21 (target_link_libraries):
  Target "kcm_sierrabreezeenhanceddecoration" links to:

    Qt6::GuiPrivate

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



-- Generating done (0.1s)
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY


CMake Generate step failed.  Build files cannot be regenerated correctly.
```

### Additional context

Upstream issue https://github.com/kupiqu/SierraBreezeEnhanced/issues/149 has been fixed in https://github.com/kupiqu/SierraBreezeEnhanced/pull/140.

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 7.0.0-cachyos-lto, NixOS, 25.11 (Xantusia), 25.11.20260413.7e495b7`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Determinate Nix 3.17.3) 2.33.3`
 - nixpkgs: `/nix/store/74cc2id747d3rgf9hg4d6ip1xqf6p88h-source`

### Notify maintainers

@A1ca7raz

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+label%3A%220.kind%3A+build+failure%22).
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/data/themes/kwin-decorations/sierra-breeze-enhanced/default.nix
+++ b/pkgs/data/themes/kwin-decorations/sierra-breeze-enhanced/default.nix
@@ -6,6 +6,7 @@
   wrapQtAppsHook,
   kwin,
   lib,
+  fetchpatch2,
 }:
 stdenv.mkDerivation rec {
   pname = "sierra-breeze-enhanced";
@@ -17,6 +18,13 @@ stdenv.mkDerivation rec {
     rev = if version == "2.1.1" then "V.2.1.1" else "V${version}";
     hash = "sha256-7mQnJCQr/zm9zEdg2JPr7jQn8uajyCXvyYRQZWxG+Q8=";
   };
+
+  patches = [
+    (fetchpatch2 {
+      url = "https://github.com/kupiqu/SierraBreezeEnhanced/pull/155.patch";
+      hash = "sha256-taj7AqiewiUU4wOTKPgfvua0rDuQ8ssH+/maxz/A3G8=";
+    })
+  ];
 
   nativeBuildInputs = [
     cmake

